lib/elixir/lib/kernel.ex

```diff
@@ -4295,6 +4295,28 @@ defmodule Kernel do
   Note that `\\` can't be used with anonymous functions because they
   can only have a sole arity.
 
+  ### Keyword lists with default arguments
+
+  Functions containing many arguments can benefit from using `Keyword`
+  lists to group and pass attributes as a single value.
```

Comment:
[the guide](https://elixir-lang.org/getting-started/keywords-and-maps.html) focuses on using keyword lists to pass a group of options - not just multiple attributes.

The current change sounds like "oh, you have 5 attributes? you can use one keyword list to pass them at once!" *blindly*. I think the code is okay, but this sentence should be updated to clarify the use case.

Instead of explaining using keyword list for grouping options, I think it's better to show how `Keyword.merge/2` can be used to implement the default arguments for keyword list.

.. then we may include `Map.merge/2` example for map arguments.. under the existing section (Default arguments) as more implementation examples, instead of adding too specific section here.